lib/Model/DataManagementPlan/DataManagementPlan.hs

```diff
@@ -24,3 +29,37 @@ instance Eq DataManagementPlan where
     _dataManagementPlanUuid a == _dataManagementPlanUuid b &&
     _dataManagementPlanQuestionnaireUuid a == _dataManagementPlanQuestionnaireUuid b &&
     _dataManagementPlanFilledKnowledgeModel a == _dataManagementPlanFilledKnowledgeModel b
+
+formatExtension :: DataManagementPlanFormat -> String
+formatExtension JSON      = ".json"
+formatExtension HTML      = ".html"
+formatExtension LaTeX     = ".tex"
+formatExtension Markdown  = ".md"
+formatExtension Docx      = ".docx"
+formatExtension ODT       = ".odt"
+formatExtension PDF       = ".pdf"
+--formatExtension RTF       = ".rtf"
+--formatExtension RST       = ".rst"
```

Comment:
Please leave the module where model datatype is defined plain. Move these helper class to seperate module next to the model module. You can inspire with `KnowledgeModel` and `KnowledgeModelAccessors`

Reply:
Done in https://github.com/DataStewardshipWizard/dsw-server/pull/6/commits/1a0ce5c65679d09b85d6cf3554b83fc294c5f5e2